src/agent/ClaimerAgent.sol

```diff
@@ -47,14 +47,15 @@ contract ClaimerAgent is Utils {
 
   uint logVerbosity;
 
-  constructor(SingleChainEnvironment _env) {
```

Comment:
the env contains all of the contracts; rather than pass it in the constructor i think it would be more consistent to reference the POOL prize vault through the env

Reply:
The easiest way to get the claimer agent to claim both the normal vault and the POOL prize vault was to create a claimer agent for each one, which is why the vault needed to be abstracted and passed in on construction.